.github/workflows/test-integration.yml

```diff
@@ -188,6 +188,7 @@ jobs:
         working-directory: python
         env:
           FELDERA_HOST: http://localhost:8080
+          TEST_FELDERA_HTTPS_TLS_CERT_PATH: test-tls/tls_test.crt
```

Comment:
I think this should be called 

`FELDERA_HTTPS_TLS_CERT`

(instead of a path, provide the cert as a string in the env variable is more common) and be usable by SDK code as well, not just tests

Reply:
we should also make sure the same functionality/env variable name is available in fda